### PR TITLE
Remove onVisibleChanged dependency

### PR DESCRIPTION
**Pull request by RensDofferhoff**
Fixes: https://github.com/jasp-stats/jasp-test-release/issues/2224
Similar:  https://github.com/jasp-stats/jaspMetaAnalysis/pull/158

### CODE REVIEW

File: inst/qml/MixedModelsBGLMM.qml [hunk 1]
```diff
@@ -49,14 +49,16 @@ Form {
...
-			onVisibleChanged:	if (!visible && count > 0) itemDoubleClicked(0);
+			visible:			active
 			name:				"dependentAggregation"
 			title:				qsTr("Number of trials")
 			singleVariable:		true
 			allowedColumns:		["scale", "ordinal"]
 
 			onEnabledChanged: if (!enabled && count > 0) itemDoubleClicked(0)
+			
+			property bool active:	family.currentText == "Binomial (aggregated)"
```
Comment (JorisGoosen) [inline inst/qml/MixedModelsBGLMM.qml:60]:
```suggestion
			property bool active:	family.currentText == qsTr("Binomial (aggregated)")
```

File: inst/qml/MixedModelsGLMM.qml [hunk 2]
```diff
@@ -49,14 +49,16 @@ Form {
...
-			onVisibleChanged:	if (!visible && count > 0) itemDoubleClicked(0);
+			visible:			active
 			name:				"dependentAggregation"
 			title:				qsTr("Number of trials")
 			singleVariable:		true
 			allowedColumns:		["scale","ordinal"]
 
 			onEnabledChanged: if (!enabled && count > 0) itemDoubleClicked(0)
+			
+			property bool active:	family.currentText == "Binomial (aggregated)"
```
Comment (JorisGoosen) [inline inst/qml/MixedModelsGLMM.qml:60]:
```suggestion
			property bool active:	family.currentText == qsTr("Binomial (aggregated)")
```

**Review by JorisGoosen — CHANGES_REQUESTED** [review]
Changes mostly look fine, didnt test it but see my requested changes

**Review by FBartos — APPROVED** [review]
at least the frequentist seems to work (the rstanarm seems to be broken on Linux)

**Review by boutinb — APPROVED** [review]

**Review by JorisGoosen — APPROVED** [review]

**Comment by RensDofferhoff** [comment]
> rstanarm

You might want to make a new issue for this!

**Pull request merged**